src/main/java/com/sparrowwallet/sparrow/control/CopyableCoinLabel.java

```diff
@@ -72,6 +72,13 @@ public void refresh(UnitFormat unitFormat, BitcoinUnit bitcoinUnit) {
     }
 
     private void setValueAsText(Long value, UnitFormat unitFormat, BitcoinUnit bitcoinUnit) {
+        if(Config.get().isHideAmounts()) {
+            setText("*****");
```

Comment:
Sorry if I wasn't clear previously. `CoinLabel.HIDDEN_AMOUNT_TEXT` should be referenced instead of `"*****"`.